src/Components/pages/Create.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import {db, fire, auth, firestore} from '../../firebase.js';
+// import {setUpRoom} from '../firebase-access.jsx'
+import Config from '../pages/Config.jsx';
+
+
+function generateGameId() {
+    let id = '';
+    const possible = '0123456789';
+    for (let i=0; i<6; i++) {
+        id += possible.charAt(Math.floor(Math.random() * 10));
+    }
+    return id;
+}
+
+class Create extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            pagetype: 'not-created',
+            gameId: '',
+        }
+        // this.createGameWrapper = this.createGameWrapper.bind(this);
+    }
+    
+    componentDidMount() {
+    }
+
+    // createGameWrapper() {
```

Comment:
there are a whole bunch of lines that are just commented out - remove those as well as the debugging statements